### PR TITLE
ID migration changes

### PR DESCRIPTION
**Pull request by bobbyiliev**
As per #399 adding the ID migration schema changes.

### CODE REVIEW

**Comment by benesch** [comment]
Impressively fast turnaround, @bobbyiliev! I won't have time to meaningfully review, so deferring review to @dehume, but looked plausible on a very quick skim.

File: examples/resources/materialize_schema/import.sh [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Schemas can be imported using the schema id:
-terraform import materialize_schema.example_schema <schema_id>
+terraform import materialize_schema.example_schema <role>:<schema_id>
```
Comment (dehume) [inline examples/resources/materialize_schema/import.sh:2]:
```suggestion
terraform import materialize_schema.example_schema <region>:<schema_id>
```

File: examples/resources/materialize_type/import.sh [hunk 2]
```diff
@@ -1,4 +1,6 @@
 # Types can be imported using the type id:
-terraform import materialize_type.example_type <type_id>
+terraform import materialize_type.example_type <region>:<type_id>
 
 # Type id and information be found in the `mz_catalog.mz_types` table
+# The region is the region where the database is located (e.g. aws/us-east-1)
+```
```
Comment (dehume) [inline examples/resources/materialize_type/import.sh:6]:
```suggestion
```
```

**Review by dehume — COMMENTED** [review]
Looks good (two minor documentation tweaks). After we confirm whether we are using the `StateUpgraders`, should be good to go

**Review by dehume — APPROVED** [review]

**Pull request merged**